src/sentry/static/sentry/app/components/forms/searchInput.tsx

```diff
@@ -56,9 +57,10 @@ const SearchReset = styled(props => <InlineSvg src="icon-circle-close" {...props
   }
 `;
 
-const SearchWrapper = styled('div')<{smaller: boolean}>`
+const SearchWrapper = styled('div')<{smaller: boolean; width: string}>`
   position: relative;
   display: inline-block;
+  width: ${p => (p.width ? p.width : 'auto')};
```

Comment:
Should we not define a `width` property if it's empty string?

Reply:
@billyvg Up to you. If width is not defined in css, the browser defaults to "auto".

Reply:
Up to you :p